cmd/options.go

```diff
@@ -1,64 +0,0 @@
-package cmd
-
-import (
-	"os"
-	"strconv"
-
-	"github.com/adshmh/meter/db"
-)
-
-const (
-	INFLUXDB_URL            = "INFLUXDB_URL"
-	INFLUXDB_TOKEN          = "INFLUXDB_TOKEN"
-	INFLUXDB_ORG            = "INFLUXDB_ORG"
-	INFLUXDB_BUCKET_DAILY   = "INFLUXDB_BUCKET_DAILY"
-	INFLUXDB_BUCKET_CURRENT = "INFLUXDB_BUCKET_CURRENT"
-
-	POSTGRES_USER     = "POSTGRES_USER"
-	POSTGRES_PASSWORD = "POSTGRES_PASSWORD"
-	POSTGRES_DB       = "POSTGRES_DB"
-	POSTGRES_HOST     = "POSTGRES_HOST"
-)
-
-type options struct {
-	db.InfluxDBOptions
-	db.PostgresOptions
-}
-
-func GatherInfluxOptions() db.InfluxDBOptions {
-	return db.InfluxDBOptions{
```

Comment:
Same as previous comments re keeping items that belong together in the same method and struct.